libs/sr/ui/src/Character/CharacterCard.tsx

```diff
@@ -22,9 +21,9 @@ export function CharacterCard({
   character: ICachedCharacter
   onClick?: () => void
 }) {
-  const calc = srCalculatorWithEntries(withMember('0', ...charData(character)))
-
-  const member0 = convert(ownTag, { et: 'own', src: '0' })
+  const calc = srCalculatorWithEntries(
```

Comment:
Since there is no other teammate, we can remove both `withMember` and `withTag`.